src/SonarScanner.MSBuild.Shim/PropertiesFileGenerator.cs

```diff
@@ -269,20 +276,35 @@ public DirectoryInfo ComputeProjectBaseDir(IList<DirectoryInfo> projectPaths)
             else if (PathHelper.BestCommonPrefix(projectPaths, pathComparer) is { } commonPrefix)
             {
                 logger.LogDebug(Resources.MSG_UsingLongestCommonBaseDir, commonPrefix.FullName, Environment.NewLine + string.Join($"{Environment.NewLine}", projectPaths.Select(x => x.FullName)));
```

Comment:
Can you edit the comment on this method?
Specifically, remove the `sonarqube/out` explanation and add that we return null.

Reply:
(maybe also mention the root logic, up to you)